app/views/returnFrequency/confirm_dates.scala.html

```diff
@@ -31,14 +31,20 @@
 
   @headings(messages("confirm_return_frequency.heading"))
 
-  <p id="p1"
-     class="form-group">
-      <strong class="bold">
-          @messages("confirm_return_frequency.newDates")
-      </strong>
-      @messages(s"return_frequency.${model.id}")
+
+  <p id="p1" class="form-group">
+        @messages("confirm_return_frequency.newDates")
+        @messages(s"return_frequency.${model.id}")
   </p>
 
+@if(annualAccounting) {
+  <p id="change-annual-accounting">@messages("confirm_return_frequency_annually.p1")</p>
+  <ul class="list list-bullet">
+     <li id="annual-accounting-bullet1">@messages("confirm_return_frequency_annually.bullet1")</li>
+     <li id="annual-accounting-bullet2">@messages("confirm_return_frequency_annually.bullet2")</li>
```

Comment:
You can use CSS selectors in the unit tests instead of having a unique id. ID is pretty much only used for making things have special styling or linking to areas of the page so as this is a normal paragraph, you should try to avoid using them

Reply:
updated with CSS selectors